balancer_wrapper.go

```diff
@@ -413,6 +413,19 @@ func (acbw *acBalancerWrapper) closeProducers() {
 	}
 }
 
+// HealthCheckOptions are the options to configure the health check producer.
+//
+// # Experimental
+//
+// Notice: This type is EXPERIMENTAL and may be changed or removed in a
+// later release.
+type HealthCheckOptions struct {
+	// Name of the gRPC service running on the server for reporting health state.
+	// If the service name is empty, client side health checking will be disabled.
+	HealthServiceName string
```

Comment:
> That is surprising to me.

The comment was wrong. There was some confusion about the service name parameter during the design review. I forgot the update the comment. I removed the line now.

Reply:
@easwars this struct is exported only because the health package needs to refer to it. 

Adding this in internal would make internal take a dependency on the `balancer` package. One option to avoid using this struct is to pass each struct member as a function parameter in `health.RegisterClientSideHealthCheckListener()`. I wanted to get your thoughts on this?

Reply:
Yes, I think that's a good idea. We don't expect to pass more params to this function other than the two that are part of this struct. So, just specifying those directly in the `RegisterClientSideHealthCheckListener` seems like a good option. And, not making it part of the public API would be a great win as well.

Reply:
Removed the exported struct and added the struct fields as params in the function.